core/lib/zksync_core/src/api_server/web3/tests/ws.rs

```diff
@@ -461,5 +471,38 @@ impl WsTest for LogSubscriptionsWithDelay {
 
 #[tokio::test]
 async fn log_subscriptions_with_delay() {
-    test_ws_server(LogSubscriptionsWithDelay).await;
+    test_ws_server(LogSubscriptionsWithDelay, None).await;
+}
+
+#[derive(Debug)]
+struct RateLimiting;
+
+#[async_trait]
+impl WsTest for RateLimiting {
+    async fn test(
+        &self,
+        client: &WsClient,
+        _pool: &ConnectionPool,
+        _pub_sub_events: mpsc::UnboundedReceiver<PubSubEvent>,
+    ) -> anyhow::Result<()> {
+        let _ = client.chain_id().await.unwrap();
```

Comment:
Nit: `let _ = ` looks redundant; the returned value is not `#[must_use]`.

Reply:
Removed.